view/frontend/web/js/jquery.card.js

```diff
@@ -0,0 +1,997 @@
+            }
+            return r.className = r.className.replace(new RegExp("(^|\\b)" + t.split(" ").join("|") + "(\\b|$)", "gi"), " ")
+        }, e.toggleClass = function(r, t, a) {
+            var n;
+            return r.length ? function() {
+                var o, i, d;
+                for (d = [], o = 0, i = r.length; o < i; o++) n = r[o], d.push(e.toggleClass(n, t, a));
+                return d
+            }() : a ? e.hasClass(r, t) ? void 0 : e.addClass(r, t) : e.removeClass(r, t)
+        }, e.append = function(r, t) {
+            var a;
+            return r.length ? function() {
+                var n, o, i;
+                for (i = [], n = 0, o = r.length; n < o; n++) a = r[n], i.push(e.append(a, t));
+                return i
+            }() : r.insertAdjacentHTML("beforeend", t)
+        }, e.find = function(r, e) {
+            return (r instanceof NodeList || r instanceof Array) && (r = r[0]), r.querySelectorAll(e)
+        }, e.trigger = function(r, e, t) {
+            var a, n;
+            try {
+                n = new CustomEvent(e, {
+                    detail: t
+                })
+            } catch (a) {
+                a,
+                (n = document.createEvent("CustomEvent")).initCustomEvent ? n.initCustomEvent(e, !0, !0, t) : n.initEvent(e, !0, !0, t)
+            }
+            return r.dispatchEvent(n)
+        }, r.exports = e
+    }).call(this)
+}, function(r, e, t) {
+    var a, n, o = [].slice;
+    n = t(3), (a = t(15)).card = {}, a.card.fn = {}, a.fn.card = function(r) {
+        return a.card.fn.construct.apply(this, r)
+    }, a.fn.extend({
+        card: function() {
+            var r, e;
+            return e = arguments[0], r = 2 <= arguments.length ? o.call(arguments, 1) : [], this.each((function() {
+                var t, o;
+                if ((o = (t = a(this)).data("card")) || (a.each(e, (function(r, t) {
+                        if (t instanceof jQuery) return e[r] = t[0]
+                    })), e.form = this, t.data("card", o = new n(e))), "string" == typeof e) return o[e].apply(o, r)
+            }))
+        }
+    })
+}, function(r, e, t) {
+    (function(e) {
+        var a, n, o;
+        t(4), n = t(1), t(8), o = t(9), a = function() {
+            var r;
+
+            function e(r) {
+                var e, t, a;
+                this.maskCardNumber = (e = this.maskCardNumber, t = this, function() {
+                    return e.apply(t, arguments)
+                }), this.options = o(!0, this.defaults, r), this.options.form ? (this.$el = n(this.options.form), this.options.container ? (this.$container = n(this.options.container), (a = n.isDOMElement(this.$container) ? this.$container : this.$container[0]).getAttribute(this.initializedDataAttr) || (a.setAttribute(this.initializedDataAttr, !0), this.render(), this.attachHandlers(), this.handleInitialPlaceholders())) : console.log("Please provide a container")) : console.log("Please provide a form")
```

Comment:
The JS is being called and inserted even when the UX option is disabled. It wouldn't be a problem, but we are getting the following error then:

> Uncaught TypeError: Cannot read property 'getAttribute' of undefined
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/191149/96657924-21029300-138f-11eb-88f2-666755ecacbf.png">

Reply:
Still got the same error on the second review.
<img width="1179" alt="image" src="https://user-images.githubusercontent.com/191149/97123495-35c09b80-1780-11eb-8deb-cf7f76d2cd5c.png">